packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -15,6 +15,7 @@
                 :label-style="normalisedLabelStyle"
                 :label-for="uniqueId"
                 :is-inline="isInline"
+                :disabled="$attrs.disabled"
```

Comment:
Does this apply a disabled attribute to the form label? If so, I'm pretty sure that's invalid HTML.

If you need to style the label based on the input being disabled, I think you'll need to either have a conditional class apply to the outer form wrapper (but you might hit scope issues with CSS modules then due to the label being a separate component), or you could pass through `$attrs.disabled` as a prop value and then set a class directly on the label element then in the sub-component.

I think you'll need to pass it through as a prop, as I don't think the label will be able to style based on the outer scope of the component.

Reply:
I'll change it to a prop. Thanks!